### DISCUSSION TITLE
Host logs and alerts

### DISCUSSION THREAD

**Discussion by kiaraho** [Security-Onion-Solutions/securityonion#10255] [2.4]
Testing out a standalone deployment of 2.4.1 with Elastic Agent installed on two Windows endpoints. `SOC > Dashboard > Host Overview` shows over 280K logs in the last 24hrs if I exclude the agent running on the Manager. However, `Kibana > Host` shows zero logs if I exclude the agent running on the Manager. Similarly, there are also zero host alerts other than those generated by the Manager itself. 

There are no issues with network logs or alerts in either the SOC or Kibana.

**Comment by kiaraho** [comment]
> Testing out a standalone deployment of 2.4.1 with Elastic Agent installed on two Windows endpoints. `SOC > Dashboard > Host Overview` shows over 280K logs in the last 24hrs if I exclude the agent running on the Manager. However, `Kibana > Host` shows zero logs if I exclude the agent running on the Manager. Similarly, there are also zero host alerts other than those generated by the Manager itself.
> 
> There are no issues with network logs or alerts in either the SOC or Kibana.

Did another test today, this time involving the generation of a malware event on a test system. Still no host logs or alerts in Kibana. However, I was able to identify the event in the Fleet logs.
 
![image](https://user-images.githubusercontent.com/5502596/235497332-eacf204e-2fe6-4191-a84a-47353d1c8c85.png)

**Reply by SasquatchSecurity** [reply]
Same issue. No Sysmon, PowerShell, or Win Security logs are coming across. Local log files are populated on my Windows hosts. Brand new 2.4.1 deployment in standalone mode. SO Firewall has been allowed for all RFC1918 IPs on the Elastic Agent group. Elastic Agent is running.

**Comment by SasquatchSecurity** [comment]
Another update... completely built a new 2.4 box off Rocky 9. Still having the same issue. Looks like most documents aren't having the event.module or event.category field written. Just deployed as "Standalone" and installed the Elastic Agent. No network traffic capture. SO Firewall has been adjusted properly, as documents are coming in.

![image](https://github.com/Security-Onion-Solutions/securityonion/assets/45203775/506df0eb-825e-46af-98eb-5960f6ed049b)

![image](https://github.com/Security-Onion-Solutions/securityonion/assets/45203775/ae10623d-58cc-40c9-922e-1cec96ee3461)

**Comment by dougburks** [comment]
Please try our new Beta 4 release and see if that helps:
https://blog.securityonion.net/2023/07/security-onion-24-beta-4-release-now.html
